extra_proposal/proposal.py

```diff
@@ -330,7 +329,12 @@ def run_ranges(sequence):
         # runs
         runs = self.runs()
         print("\nRuns collected: {} (total {})".format(*run_ranges(runs)))
-        if self.damnit() is not None:
+
+        try:
+            self.damnit()
+        except Exception:
```

Comment:
Nitpicking, I think we can go for a more specific exception:
```suggestion
        except FileNotFoundError:
```